foundry.toml

```diff
@@ -18,4 +18,22 @@ src = 'src'
 test = 'src'
 via_ir = true
 
-# See more config options https://github.com/foundry-rs/foundry/tree/master/config
+[rpc_endpoints]
```

Comment:
this new config allows us to configure rpc url by chain name

Reply:
cxould we be opinionated and just take an alchemy key?

Reply:
sure that makes sense